public/js/strings.json

```diff
@@ -6,6 +6,10 @@
   "callStack.expand": "Expand Rows",
   "editor.searchResults": "%d of %d results",
   "editor.noResults": "no results",
+  "editor.addBreakpoint": "Add Breakpoint",
+  "editor.removeBreakpoint": "Remove Breakpoint",
+  "editor.addConditionalBreakpoint": "Add Conditional Breakpoint",
+  "editor.editConditionalBreakpoint": "Edit Conditional Breakpoint",
```

Comment:
How are we going to keep strings in sync? It worries me a little bit that we need to maintain them in two places.